### ISSUE TITLE
Window contents disappear on window maximize/un-maximize

### ISSUE THREAD

**Issue by rasa** [okibcn/nano-for-windows#5]
In either a `cmd` (Command Prompt) window (and not using "Legacy Console" mode), or a `wt` (Windows Terminal Preview Version: 1.17.1023) "Windows PowerShell" tab, and running
```
C:\Users\ross\scoop\apps\nano\7.2-8-10163.1\nano.exe filename
```
and then clicking the maximize button in the upper right of the window, the window content goes black.

If I type `^L`, the content reappears. The window also goes blank when I click the maximize button again to restore the window to its original size.

Running
```
"C:\Program Files\Git\usr\bin\nano.exe" filename
```
in either `cmd` or `wt` window, the contents of the window are redrawn correctly when the window is maximized, or un-maximized.

I've been trying to fix this by changing the Windows Terminal settings, but haven't found a solution yet.

**Comment by rasa** [comment]
I did some more testing in a Windows Sandbox, and the issue occurs there too, so I know it's not just my system acting wierd.

**Comment by okibcn** [comment]
No, its not. And the problem is not related to maximize or restore the window. It seems that it is redrawing the screen only on the even resizes and odd resizes are not redrawn. I will look into it.

**Comment by okibcn** [comment]
Fixed in [v7.2-8-10163.3](https://github.com/okibcn/nano-for-windows/releases/tag/v7.2-8-10163.3).

**Comment by okibcn** [comment]
Fixed in [v7.2-8-10163.3](https://github.com/okibcn/nano-for-windows/releases/tag/v7.2-8-10163.3).
